doc/language_guide.md

```diff
@@ -328,6 +330,39 @@ WITH
 USING TreeExplainer;
 ```
 
+## Evaluation Syntax
+
+A SQLFlow prediction statement consists of a sequence of select, predict, and using clauses.
+
+```sql
+SELECT select_expr [, select_expr ...]
+FROM table_references
+  [WHERE where_condition]
+  [LIMIT row_count]
+TO EVALUATE model_table_reference
+[WITH
+  attr_expr [, attr_expr ...]]
+INTO evaluate_result_table;
+```
+
+The select statement syntax is the same as the select statement syntax in the training syntax. When doing the evaluation, you should use a dataset that is not used for training to evaluate the model's performance when dealing with "new" data. The evaluation result will be saved in `evaluate_result_table` when the evaluation job completes.
+
+### Evaluate and Into Clause
+
+The *evaluate clause* describes which model will be used to evaluate, what metrics will be used for evaluation, and which table will be used to save the result.
```

Comment:
*evaluate clause* => `TO EVALUATE` clause

describes which model => specifies the model

will be used to evaluate => to evaluate